src/project.rs

```diff
@@ -263,7 +394,77 @@ impl FuzzProject {
         cmd.arg("-minimize_crash=1")
             .arg(format!("-runs={}", tmin.runs))
             .arg(&tmin.test_case);
-        exec_cmd(&mut cmd).with_context(|| format!("could not execute command: {:?}", cmd))?;
+
+        let old_artifacts = self.get_artifact_paths(&tmin.target)?;
+
+        let mut child = cmd
+            .spawn()
+            .with_context(|| format!("failed to spawn command: {:?}", cmd))?;
+        let status = child
+            .wait()
+            .with_context(|| format!("failed to wait on child process for command: {:?}", cmd))?;
+        if !status.success() {
+            eprintln!("\n{:─<80}\n", "");
+            return Err(anyhow!("Command `{:?}` exited with {}", cmd, status))
+                .with_context(|| {
+                    "Test case minimization failed.\n\
+                     \n\
+                     Usually this isn't a hard error, and just means that libfuzzer\n\
+                     doesn't know how to minimize the test case any further while\n\
+                     still reproducing the original crash.\n\
+                     \n\
+                     See the logs above for details."
+                })
+                .into();
+        }
+
+        // Find and display the most recently modified artifact, which is
+        // presumably the result of minification. Yeah, this is a little hacky,
+        // but it seems to work. I don't want to parse libfuzzer's stderr output
+        // and hope it never changes.
+        let mut new_artifacts: Vec<_> = self
+            .get_artifact_paths(&tmin.target)?
+            .into_iter()
+            .filter(|a| !old_artifacts.contains(a))
+            .collect();
+        new_artifacts.sort_by_key(|p| {
+            p.metadata()
+                .and_then(|m| m.modified())
+                .unwrap_or(std::time::SystemTime::UNIX_EPOCH)
+        });
+
+        if let Some(artifact) = new_artifacts.pop() {
```

Comment:
Should we just iterate here, printing the latest one last? Or have a flag that lets you do that.

Reply:
Maybe we could add a flag, but there is going to be one file for every reduction from the original to the minimal, which seems like it could be a very long path. Generally, one only cares about the final result.

Reply:
Sounds okay to me!